app/views/content/events/what-happens-at-events-transcript.md

```diff
@@ -0,0 +1,104 @@
+---
+title: What happens at a Get Into Teaching event video transcript
+description: Full transcript for the video 'What happens at a Get Into Teaching event' video.
+---
+
+**Speaker 1 - Georgie Butler:** 
+
+Here we are in Manchester for the first Get Into Teaching event of the series.
+If you’ve ever wondered what happens at a Get Into Teaching event, let’s go take a look.
+
+[Video shows people and stalls at the event, including the 'advice zone' and 'teacher hub']
+
+**Speaker 1:** 
+
+These Get Into Teaching events will provide information, advice and support to anyone looking into a career in teaching. 
+
+**Speaker 2 - Amanda Bartley, On-campus Marketing Manager, Department for Education:** 
+
+The purpose of this event is to get people like you into teaching, to inspire the next generation. Teachers are fundamental to our society, so we want to help you on your journey to becoming a teacher.
+
+[Video shows teaching presentations and stalls in the 'training provider zone' at the event]
+
+**Speaker 1:**
+
+It’d be really interesting to hear how you found today’s event.
+
+**Attendee 3:** 
```

Comment:
Technically, wouldn't they be Attendee 1? In which case, all the attendee numbers would need decreasing by 2?

But if you think this is too confusing to have a Speaker 1 and an Attendee 1, let's leave as is!